osgar/logger.py

```diff
@@ -106,7 +106,7 @@ def write(self, stream_id, data, dt=None):
         with self.lock:
             if dt is None:
                 # by defaut generate timestamps automatically
-                dt = datetime.datetime.utcnow() - self.start_time
+                dt = datetime.datetime.now(datetime.timezone.utc) - self.start_time
```

Comment:
what is the difference between old `utcnow()` and new `now(datetime.timezone.utc)`?

Reply:
`utcnow()` creates a datetime object that is timezone naive (=does not have information about which timezone it represents) while `now(datetime.timezone.utc)` knows its time is UTC. This helps later on when you want to call `walltime.timestamp()`. Timezone naive datetime assumes it represents local time therefore when converting to timestamp you get local one and not UTC. That fails especially when you would like to compare timestamps on computers that have different timezones.